client/gutenberg/extensions/map/mapbox-map-formatter/index.js

```diff
@@ -4,20 +4,20 @@
 export function mapboxMapFormatter( mapStyle, mapDetails ) {
```

Comment:
We can remove this "/* TODO: Replace with styles created in A8C-owned account */" line now

Reply:
Ah sorry, this was meant to be comment for line 3

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/28729/commits/ab4af733c103ca97b88fc6875e413d65df8ad574.